src/App.js

```diff
@@ -1,30 +1,24 @@
 import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import SignIn from './pages/SignIn';
+import SignUp from './pages/SignUp';
+import Footer from './components/Footer';
+import Header from './components/Header';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
```

Comment:
Hey @marinacod I don't think we want to use the `react-router-dom` package here.

Once the backend gets some basic routes set up, we will be using those for route switching between pages.